service.go

```diff
@@ -83,6 +83,12 @@ func (s *Service) SearchCerts(clusterID string) (AssetsBundle, error) {
 	return assetsBundle, nil
 }
 
+// SearchEncryptionKey watches for encruption key secrets of a cluster  and returns it as
+// assets bundle.
+func (s *Service) SearchEncryptionKey(clusterID string) (string, error) {
+	return s.SearchEncryptionKeyForComponent(clusterID, "k8s-encryption")
```

Comment:
Is there now no conflict because we use a different name?

Reply:
yes exactly